sources/packages/backend/apps/api/src/app.system-access.module.ts

```diff
@@ -2,78 +2,48 @@ import { Module } from "@nestjs/common";
-import { MSFAARequestService } from "./esdc-integration/msfaa-integration/msfaa-request.service";
-import { MSFAAIntegrationService } from "./esdc-integration/msfaa-integration/msfaa-integration.service";
-import { MSFAAResponseService } from "./esdc-integration/msfaa-integration/msfaa-response.service";
-import { ECertFileHandler } from "./esdc-integration/e-cert-integration/e-cert-file-handler";
-import { ECertFullTimeIntegrationService } from "./esdc-integration/e-cert-integration/e-cert-full-time-integration/e-cert-full-time-integration.service";
-import { ECertPartTimeIntegrationService } from "./esdc-integration/e-cert-integration/e-cert-part-time-integration/e-cert-part-time-integration.service";
-import { ECertFullTimeFileHeader } from "./esdc-integration/e-cert-integration/e-cert-full-time-integration/e-cert-files/e-cert-file-header";
-import { ECertFullTimeFileFooter } from "./esdc-integration/e-cert-integration/e-cert-full-time-integration/e-cert-files/e-cert-file-footer";
-import { ECertPartTimeFileHeader } from "./esdc-integration/e-cert-integration/e-cert-part-time-integration/e-cert-files/e-cert-file-header";
-import { ECertPartTimeFileFooter } from "./esdc-integration/e-cert-integration/e-cert-part-time-integration/e-cert-files/e-cert-file-footer";
-import { DisbursementReceiptProcessingService } from "./esdc-integration/disbursement-receipt-integration/disbursement-receipt-processing.service";
-import { DisbursementReceiptIntegrationService } from "./esdc-integration/disbursement-receipt-integration/disbursement-receipt-integration.service";
-import { DisbursementReceiptRequestService } from "./esdc-integration/disbursement-receipt-integration/disbursement-receipt-request.service";
-import { FedRestrictionProcessingService } from "./esdc-integration/fed-restriction-integration/fed-restriction-processing.service";
-import { FedRestrictionIntegrationService } from "./esdc-integration/fed-restriction-integration/fed-restriction-integration.service";
-import { WorkflowClientService, SequenceControlService } from "@sims/services";
-// todo: once all integration are moved. remove sshservice.
-import { SshService } from "@sims/integrations/services";
 import {
   SFASIndividualService,
   SFASApplicationService,
   SFASPartTimeApplicationsService,
 } from "@sims/services/sfas";
-import { IER12IntegrationModule } from "@sims/integrations/institution-integration/ier12-integration/ier12-integration.module";
 
 @Module({
-  imports: [AuthModule, SINValidationModule, IER12IntegrationModule],
-  controllers: [
-    SINValidationSystemAccessController,
-    ApplicationSystemAccessController,
-    MSFAAIntegrationSystemAccessController,
-    ECertIntegrationSystemAccessController,
-    FedRestrictionsIntegrationSystemAccessController,
-  ],
+  imports: [AuthModule],
+  controllers: [ApplicationSystemAccessController],
```

Comment:
This controller is the last one and it has only the `archiveApplications` process now.
I am not sure if we have another ticket to do this but I was not able to find a Queue for it, so I am assuming it is a gap.
If I am right, can we have a ticket created to add this to the queue list and create the scheduler?
@JasonCTang FYIA

Reply:
Also if archiveApplications is the last in the module, then we can remove it from the module (having a ticket created to create a queue for archive) and leave it orphaned. By doing so we can delete the system access module and do the clean up.

Reply:
yes, we missed it. we can create a sticker for that